frameworks/engineering/backend.md

```diff
@@ -201,22 +202,23 @@ topics:
           - "Debugs production issues"
           - "Considers metrics when building a new system"
         exampleCriteria:
-          - criteria: "Writes idiomatic Go code"
+          - criteria: "Writes idiomatic code"
             examples:
               - "Follows the guidelines in Effective Go (https://golang.org/doc/effective_go.html), avoids using patterns that are idiomatic in other languages"
           - criteria: "Mitigates race conditions and partial failure states in distributed systems"
             examples:
               - "Uses locking where appropriate, writes code that is “self-healing” when dealing with distributed failure (e.g. data needs to be written in two or more services)"
-          - criteria: "Implements complex RPC handlers"
+          - criteria: "Implements complex RPC handlers/scripts"
             examples:
               - "Implements complex business logic, orchestrates multi-stage processes, deals with idempotency or distributed data sources"
+              - "Implements complex scripts to make cluster-wide changes to production (e.g. rolling out a change to the service mesh)"
       - level: 4
         criteria:
           - "Writes code that serves as a definitive example for new engineers"
           - "Leads the refactoring (de-cruftification) of complex systems"
           - "Debugs complex production issues at speed"
           - "Identifies and fixes security vulnerabilities"
-          - "Identifies and fixes performance bottlenecks in individual services"
+          - "Identifies and fixes performance bottlenecks in individual systems/services"
```

Comment:
How do we define 'system' in this context? Elsewhere in this framework 'system' is used to mean 'collection of multiple services'

Reply:
By system we meant other applications which might not necessarily be services (at least in the Monzo definition). This includes things like Kubernetes, Cassandra, NSQ etc.

Reply:
@suhailpatel thanks, elsewhere in the framework we use 'system' to mean 'collection of services' (rightly or wrongly) if it's fine for those two uses to sit side by side that's fine. Otherwise I would suggest 'components' or something instead.